Makefile

```diff
@@ -49,29 +49,30 @@ extract_translations: ## extract strings to be translated, outputting .mo files
 
 fake_translations: extract_translations dummy_translations compile_translations ## generate and compile dummy translation files
 
-COMMON_CONSTRAINTS_TXT=requirements/common_constraints.txt
-.PHONY: $(COMMON_CONSTRAINTS_TXT)
-$(COMMON_CONSTRAINTS_TXT):
-	wget -O "$(@)" https://raw.githubusercontent.com/edx/edx-lint/master/edx_lint/files/common_constraints.txt || touch "$(@)"
-
 upgrade: export CUSTOM_COMPILE_COMMAND=make upgrade
-upgrade: $(COMMON_CONSTRAINTS_TXT)  # update the requirements/*.txt files with the latest packages satisfying requirements/*.in
+
+# Define PIP_COMPILE_OPTS="-v" to get more information during make compile-requirements.
+compile-requirements: export CUSTOM_COMPILE_COMMAND=make upgrade
+compile-requirements: ## Re-compile requirements/*.in files to *.txt.
 	pip install -r requirements/pip-tools.txt
-	pip-compile --upgrade --allow-unsafe --rebuild -o requirements/pip.txt requirements/pip.in
-	pip-compile --no-emit-trusted-host --no-emit-index-url --rebuild --upgrade -o requirements/pip-tools.txt requirements/pip-tools.in
+	pip-compile $(COMPILE_OPTS) --allow-unsafe requirements/pip.in
```

Comment:
Where have we defined `COMPILE_OPTS` in the file? or are we passing an empty placeholder instead?